bbsr/sct-tests/SecureBoot/BlackBoxTest/VariableUpdatesBBTest.c

```diff
@@ -233,7 +233,7 @@ VariableUpdatesTestCheckpoint1 (
   // Test KEK update with unsigned data, expect security violation
   //
 
```

Comment:
Add a comment with the properties of this new file. 

- Size of file
- Its signature properties